### ISSUE TITLE
Unable to deploy a project with Werkzeug >= 2.0

### ISSUE THREAD

**Issue by rjschave** [zappa/Zappa#1101]
## Context

After creating a new virtual environment and installing my project dependencies, including Zappa 0.54.1, I am no longer able to deploy my project.  

My Django project does not use Werkzeug, but Werkzeug 2.0.2 gets installed by Zappa.  After downgrading to Werkzeug<2.0.0, I am able to deploy my project again.

Updating Zappa to 0.54.1 from an older version that installed Werkzeug 1.0.1 still works because the version of Werkzeug is left unchanged.

I have confirmed this behavior with both Python 3.6 and Python 3.8 and with MacOS 10.15.7 and MacOS 12.1.

## Expected Behavior

Your updated Zappa deployment is live!: 

## Actual Behavior

Error: Warning! Status check on the deployed lambda failed. A GET request to '/' yielded a 502 response code.

Digging into the Cloud Watch logs, I see the error as described in #1087.

## Possible Fix
Specify a specific version of Werkzeug in Zappa dependencies.  Werkzeug 1.0.1 works for me.

## Steps to Reproduce
Install Zappa 0.54.1 into a new virtual environment.  Attempt to deploy your project.

**Comment by sagar1025** [comment]
I ran into the same issue. I had to manually uninstall and reinstall these versions of Zappa and Werkzeug:

`zappa==0.54.0`
`Werkzeug==2.0.2`

Dont forget to update the `requirements.txt`!

**Comment by rjschave** [comment]
Confirmed rebuilding my virtual environment resolved the issue.

I don't see that anything changed with Werkzeug since I opened the issue.   It does look like the Python 3 wheel for Zappa changed, even though the version number is the same (see https://github.com/zappa/Zappa/issues/1102#issuecomment-1009304452 for more info).

Looking at PyPI, I see the source for 0.54.1 was released on 11/11/2021.  An updated wheel for 0.54.1 was released on 1/7/2022.  I downloaded both, diffed the code, and confirmed the source in the wheel is different.  I reviewed the changes, but it's not obvious to me that these changes are the reason why I can now deploy with Werkzeug version 2.0.2.
